game-core/src/main/java/games/strategy/triplea/delegate/battle/AirBattle.java

```diff
@@ -705,7 +705,6 @@ public void execute(final ExecutionStack stack, final IDelegateBridge bridge) {
                       defendingUnits,
                       attackingUnits,
                       false,
-                      new ArrayList<>(),
```

Comment:
Does the list need to be modifiable? If not, `List.of()` is the conventional preference for empty lists.

Reply:
This list was removed.